### DISCUSSION TITLE
Возможность задать данные для авторизации

### DISCUSSION THREAD

**Discussion by endofreal** [hufrea/byedpi#119] [Ideas]
Есть идея сделать для себя данный прокси вездесущим и высветить в интернет. Без авторизации это выглядит совсем смешно. Можно ли добавить этот функционал?

**Comment by solodyagin** [comment]
Может быть получится использовать цепочку из прокси 3proxy => ByeDPI (https://3proxy.ru/howtor.asp#CHAIN) ?
3proxy поддерживает авторизацию.
